Dockerfile

```diff
@@ -1,36 +1,33 @@
-FROM node:16 AS node-build
+FROM registry.suse.com/bci/nodejs:14 AS node-build
 WORKDIR /build
 # skip the hack/get_version_from_git.sh execution in the frontend build
 ENV VERSION=""
 # first we only add what's needed to run the makefile
 ADD Makefile /build/
 # then we add what's needed just to install node packages so that dependencies can be cached in a dedicate layer
 ADD web/frontend/package.json web/frontend/package-lock.json /build/web/frontend/
-RUN make web-deps
+RUN zypper -n in make && make web-deps
 # only as last step we add the web frontend sources, this way changing these doesn't retrigger the slow npm install
 ADD web/frontend /build/web/frontend/
 RUN make web-assets
 
-FROM golang:1.16 as go-build
+FROM registry.suse.com/bci/golang:1.16 as go-build
 WORKDIR /build
 # we add what's needed to download go modules so that dependencies can be cached in a dedicate layer
 ADD go.mod go.sum /build/
 RUN go mod download
 ADD . /build
 COPY --from=node-build /build /build
-RUN make build
+RUN zypper -n in make git && make build
 
-FROM python:3.7-slim AS trento-runner
-RUN ln -s /usr/local/bin/python /usr/bin/python \
-    && /usr/bin/python -m venv /venv \
-    && /venv/bin/pip install 'ansible~=4.6.0' 'requests~=2.26.0' 'rpm==0.0.2' 'pyparsing~=2.0' \
-    && apt-get update && apt-get install -y --no-install-recommends \
-      ssh \
-    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
-    && rm -rf /var/lib/apt/lists/*
+FROM registry.suse.com/bci/python:3.9 AS trento-runner
+RUN /usr/bin/python3 -m venv /venv \
+    && /venv/bin/pip install 'ansible~=4.6.0' 'ara~=1.5.7' 'rpm==0.0.2' 'pyparsing~=2.0' \
```

Comment:
it was requests, now its ara, is this an intentional change?

Reply:
Really true! This is an error. It should be `requests`!